### ISSUE TITLE
mongotail 3.0.0 not compatible with res-address 2.0.0

### ISSUE THREAD

**Issue by mtskillman** [mrsarm/mongotail#41]
![image](https://user-images.githubusercontent.com/55630669/160896924-646b1f11-2d70-4d88-9215-9f1c3b04c6c1.png)

res-address 2.0.0:
`def get_res_address(address):
    """
    <REDACTED>
    :return: a tuple with ``(scheme, host, port, resource, query, username, password)``. <REDACTED>
    """`

mongotail 3.0.0:
```
def connect(address, args):
    ...
    try:
        host,  port, dbname = get_res_address(address)
```

it appears issue started in commit 1ef2c825984050ad5d296d9be55831f312fb625f of the res-address project, in which get_res_address began returning more than 3 values

**Comment by mrsarm** [comment]
@mtskillman thanks for your report, good catch ! I didn't see that coming when I released today the res-addess library, I was going to release the new Mongotail 3.1.0 version that works with res-address 2.0, but yes that makes older versions of Mongotail incompatible, that's why Mongotail 3.1 will limit the range of versions of the libraries depends on to not have the same problems in the future.

I going to release Mongotail 3.1.0 in a few moment so you can use that version today.

**Comment by mrsarm** [comment]
@mtskillman I just released mongotail [3.1.0](https://github.com/mrsarm/mongotail/releases/tag/3.1.0) with new features and fixes. You should be able to update executing `pip3 install -U mongotail`.
